ph5/utilities/ph5validate.py

```diff
@@ -53,31 +55,13 @@ class PH5ValidateException(Exception):
 
 
 class PH5Validate(object):
-    def __init__(self, ph5API_object, ph5path,
-                 level, outfile):
+    def __init__(self, ph5API_object, ph5path):
```

Comment:
Does removing outfile, remove the capability to have output to a file instead of stdout?

Reply:
Changing the call signature here?
Is this class used anywhere else?

Reply:
> Does removing outfile, remove the capability to have output to a file instead of stdout?

No, this outfile is to setup log file. I setting log file at line https://github.com/PIC-IRIS/PH5/blob/dca1c146e90848bcf1f3d415fe1440a6f741f769/ph5/utilities/ph5validate.py#L842
You can see that unittest still be able to check the content of ph5_validate.log as before changing logging configuration.

Reply:
> Is this class used anywhere else?

This is a validating tool, it isn't used anywhere else except for its unittest.

Reply:
What??!!?!
This was only written to be tested?

Reply:
This is used to validate ph5 before being archived!!!!

Reply:
> This was only written to be tested?

No I means this is call when this ph5_validate tool is used from command line. This is not call from other files in PH5.